### PR TITLE
Force local package install

### PR DESCRIPTION
**Pull request by soupette**
Signed-off-by: soupette <cyril@strapi.io>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "scripts": {
     "analyze:bundle": "lerna run analyze:bundle --stream",
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --force-local",
```
Comment (mfrachet) [inline package.json:9]: force local should not be necessary 🤔 . design-system and icons are working together without that specific need

File: examples/app/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "dependencies": {
     "@craco/craco": "^6.1.1",
-    "@strapi/design-system": "../../packages/strapi-design-system",
+    "@strapi/design-system": "0.0.1",
```
Comment (mfrachet) [inline examples/app/package.json:7]: Also, in order to make the dependency resolve, have you used:

```sh
$ npx lerna add @strapi/design-system --scope=strapi-ds-app-example
```

???

Comment (soupette) [reply examples/app/package.json:7]: Nope! I'll update my PR

**Review by mfrachet — APPROVED** [review]
LGTM, thank you 😊

**Pull request merged**